seed/seed.json

```diff
@@ -25,7 +25,7 @@
             ],
             "filter": {
                 "min_version": "91.0.4472.114",
```

Comment:
this is wrong version. it should be `<chrome>.<brave>`, see other entries @antonok-edm @kjozwiak

Reply:
@moritzhaller maybe we could add a version validation step?

Reply:
ah, so it's the major version of Chromium followed by all 3 version sections of Brave?

At a first glance it looked just like a full Chromium version, so that's what I used. Probably would be good to add this information to the README at the very least.